__tests__/app.test.js

```diff
@@ -30,20 +30,37 @@ describe('/api/topics', () => {
 
 describe('/api/articles/:article_id', () => {
     test('GET request responds with an array of topic objects, each of which has slug and description properties', () => {
-        const article_id = 1
         return request(app)
         .get('/api/articles/1')
         .expect(200)
         .then(( { body }) => {
-            expect(body).toMatchObject({
+            expect(body.article).toMatchObject({
                 author: expect.any(String),
                 title: expect.any(String),
-                article_id: article_id,
+                article_id: 1,
                 body: expect.any(String),
                 topic: expect.any(String),
                 created_at: expect.any(String),
                 votes: expect.any(Number)
             })
```

Comment:
Well done for the change - would still take this specificity further. With your test data you are able to see exactly what details this article has so you can also assert for values of other properties